en/services/high_latency.md

```diff
@@ -1,27 +1,29 @@
 
 ## Heartbeat/Routing
 
 High latency links are expensive.
 In order to reduce traffic to the bare minimum, some of the fundamental assumptions of MAVLink are explicitly broken:
-- [HEARTBEAT](../messages/common.md#HEARTBEAT) messages are not emitted on the channel (either by the autopilot or GCS).
+- [HEARTBEAT](../messages/common.md#HEARTBEAT) messages are not emitted on the channel by the autopilot. The GCS will continue to send the heatbeat to the autopilot though.
 
-  > **Note** The heartbeat is used to build MAVLInk routing tables between channels.
+  > **Note** The heartbeat is used to build MAVLink routing tables between channels.
     Commands addressed specifically to the high latency component may not be routed from another channel (i.e. you can connect to the component from a GCS directly, but not via a MAVLink router).
-- Broadcast messages from the network should not be automatically sent over the high latency channel.
+- Only the [COMMAND_LONG](../messages/common.md#COMMAND_LONG) and [MAV_CMD_ACK](../messages/common.md#MAV_CMD_ACK) messages should be sent over the high latency channel.
+- The GCS should send the [HEARTBEAT](../messages/common.md#HEARTBEAT) message, in order to allow for loss-of-datalink failsafes to work.
+- Ground stations should not request full sets of waypoints or parameters in order to prevent congestion.
 
 The other rules are essentially the same but there are some implications of the above changes:
-- Broadcast messages from the high latency channel should be routed to other nodes on the network as usual.
+- Messages from the high latency channel should be routed to other nodes on the network as usual.
   Note that this in reality most systems on a high latency network **only** send `HIGH_LATENCY2`.
-- Addressed messages should be sent over the high latency channel (in both directions) in accordance with the normal routing rules.
-  In practice the lack of `HEARTBEAT` means that addressed messages are unlikely to arrive, and hence be sent.
+- Most addressed messages will not be sent over the high latency link.
+- Parameters, waypoints, geofences and rally points cannot be updated or changed over a high latency channel.
 
-The implication is that while all components on a MAVLink network will get [HIGH_LATENCY2](#HIGH_LATENCY2) updates, only the directly connected GCS (or other component) will be able to command the vehicle.
+The implication is that while all components on a MAVLink network will get [HIGH_LATENCY2](#HIGH_LATENCY2) updates, any the directly connected GCS (or other component) will be able send COMMAND_LONG commands to the vehicle.
```

Comment:
Grammar error injected at "any the directly". Again, "command protocol messages".

Reply:
Fixed